src/tests/landingpage.test.js

```diff
@@ -4,8 +4,7 @@ import React from 'react';
 import Landingpage from '../components/Landingpage/Landingpage';
 import Contact from '../components/Contact/Contact';
 import Services from '../components/Landingpage/Services';
-
-
+import Footer from '../components/Footer/Footer'
```

Comment:
This Footer imported in not available. The footer you are using does not have a capital letter. It might cause some errors.